starter-code/pizza.js

```diff
@@ -1 +1,43 @@
 // Write your Pizza Builder JavaScript in this file.
+$(document).ready(function() {
+  var pep = 1;
```

Comment:
this is a great idea. Might be better to hold it all in an object:
`
var priceList = 
{
pep : 1,
mush: 1,
...
etc.
}
`